modules/tracker/models/pdo/TrendModel.php

```diff
@@ -58,6 +59,10 @@ public function getMatch($producerId, $metricName, $configItemId, $testDatasetId
             $sql->where('test_dataset_id = ?', $testDatasetId);
         }
 
+        if (!is_null($unit) && ($unit !== '')) {
```

Comment:
@jamiesnape 

I'm not sure if you are suggesting replace both parts of the test or only the empty string.

I'm changing this test to 

    if ($unit !== false) {

if that's ok with you.